app/src/main/java/com/example/foodjuju/ui/calendar/CalendarViewModel.kt

```diff
@@ -7,7 +7,7 @@ import androidx.lifecycle.ViewModel
 class CalendarViewModel : ViewModel() {
 
     private val _text = MutableLiveData<String>().apply {
-        value = "This is dashboard Fragment"
+        value = "This is the dashboard fragment."
     }
-    val text: LiveData<String> = _text
+   internal val text: LiveData<String> = _text
```

Comment:
Yes, this makes more sense as internal... unless it will be accessed outside the module.